src/test/java/com/google/collegeplanner/CourseListTest.java

```diff
@@ -62,15 +66,15 @@ public void after() {
   @Test
   public void servletResponseHasCourses() throws Exception {
     CourseListServlet servlet = new CourseListServlet();
-    servlet.doGet(null, mockedResponse);
+    servlet.doGet(mockedRequest, mockedResponse);
 
     JSONObject responseObj = (JSONObject) parser.parse(stringWriter.toString());
     JSONArray coursesDetailed = (JSONArray) responseObj.get("courses_detailed");
 
     // Tests that courses_detailed exists
     Assert.assertNotNull(coursesDetailed);
     // Checks that the correct number of JSON Objects are contained
-    Assert.assertEquals(coursesDetailed.size(), 30);
+    Assert.assertEquals(coursesDetailed.size(), 29);
```

Comment:
why did this change? Is this not hermetic? I think you might need to do what Savion did in his PR, and mock out the actual underlying HTTP call.

Reply:
We changed the function of the CourseListServlet to not return a specific number of courses, but to return all courses in a department, which is why the number changed -- there are only 29 courses in the 'AASP' department. Other than those two additions, I didn't change any of Savion's  tests.

Reply:
I didn't mock the underlying http call. I just mocked ApiUtil. We concluded that mocking the underlying http call was too complicated.

Reply:
Good point, thanks for the clarification Savion. So for this function, we should mock out ApiUtil too.